cmd/ci-operator/main.go

```diff
@@ -1114,6 +1129,12 @@ func (o *options) writeJUnit(suites *junit.TestSuites, name string) error {
 		return nil
 	}
 	suites.Suites[0].Name = name
+	sort.Slice(suites.Suites, func(i, j int) bool {
+		return suites.Suites[i].Name < suites.Suites[j].Name
+	})
+	for i := range suites.Suites {
+		sortSuite(suites.Suites[i])
+	}
```

Comment:
Will this affect how real jobs are displayed?

Reply:
No, it just sorts the XML.

Reply:
I'm pretty sure most jUnit viewers will respect that sort though. Order matters in XML. Which means that Spyglass will show probably start to show the items sorted instead of in the run order.